tests/api.c

```diff
@@ -51,11 +51,18 @@ int myoptind = 0;
 char* myoptarg = NULL;
 
 
+#ifndef WOLFSSH_NO_ASSERT_FAIL_ABORT
```

Comment:
This name, WOLFSSH_NO_ASSERT_FAIL_ABORT, is kind of verbose and confusing. Does it just control if WABORT maps to abort? How about WOLFSSH_NO_WABORT?